akka-sample-remote-java/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>akka-sample-remote-java</artifactId>
+  <groupId>com.typesafe.akka.samples</groupId>
+  <name>Akka Main in Java</name>
```

Comment:
This is not the 'Main' example but the 'Remote' example, perhaps update the name?